scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SubmitHandler.java

```diff
@@ -91,6 +91,7 @@ public void run() {
         for (InternalJob job : jobs) {
             if (job != null) {
                 service.getJobs().jobSubmitted(job, timingLogger);
+                frontendState.jobSubmitted(job, ident);
```

Comment:
I believe you need to add
if (forntendState != null) {
}
This is for unit tests